package.json

```diff
@@ -28,8 +28,8 @@
         "test": "./node_modules/.bin/mocha "
     },
     "dependencies": {
-        "axios": "0.21.1",
-        "bcryptjs": "^2.4.3",
+        "axios": "^0.21.1",
+        "bcryptjs": "^5.0.0",
```

Comment:
we still support nodejs 8 - this change drops support for 8 - so we can't go there until we do a major update. See https://discourse.nodered.org/t/bcrypt-update-from-3-0-6-to-5-0-0/32693

Reply:
This dependency is `bcryptjs` **not** `bcrypt`. There is *nothing* to change about `bcryptjs`.

Your point is valid about `bcrypt` - but this is *not* that module being changed here - but the change is still invalid as per my other review comment.